packages/paste-website/src/pages/content/index.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Guideline Usage
+description: An overview to help you navigate product content guidance and a look at what's coming soon.
+---
+
+import {graphql} from 'gatsby';
+import {Link} from 'gatsby';
+import {Blockquote} from '../../components/Blockquote';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+import {DoDont, Do, Dont} from '../../components/DoDont';
+import Changelog from '@twilio-paste/icons/CHANGELOG.md';
+import {Anchor} from '@twilio-paste/anchor';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/content/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<p>{props.pageContext.frontmatter.description}</p>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings}/>
+
+<content>
+
+<Callout variant="warning">
+  <CalloutTitle as="h4">Early Access! Early Access! Early Access!</CalloutTitle>
+  <CalloutText>These guidelines are still in development and are not yet complete.
+  
+  If you can't see what you need here, you've found an edge-case or exception, or you have other comments, reach out to Caroline Tecks on Slack.</CalloutText>
+</Callout> 
+
+**Use the guidance in the way that makes  most sense to you.** 
+These content pages should help you to feel more confident in writing concise, clear, and consistent product content.
+
+You might read [Voice and tone](//content/voice-and-tone) once for foundational understanding, while you might bookmark [Product style guide](//content/product-style-guide) for ease of reference.
```

Comment:
There are a couple of links that have a double `//` at the start which don't go to the desired page. You might want to do a quick pass to make sure all links are working correctly. It might only be the 2 `//content/voice-and-tone` links in this page